src/evaluator.rs

```diff
@@ -5,7 +5,7 @@ use std::process::exit;
 
 #[derive(Debug)]
 pub enum LimpValue {
-	Integer(u64),
```

Comment:
oh lol, this is pretty dumb on my part 😢

Reply:
it's a tiny issue